src/script/entity/Conversation.ts

```diff
@@ -150,7 +149,7 @@ export class Conversation {
   public readonly legalHoldStatus: ko.Observable<LegalHoldStatus>;
   public readonly localMessageTimer: ko.Observable<number>;
   public readonly messages_unordered: ko.ObservableArray<Message>;
-  public readonly messages_visible: ko.PureComputed<Message[]>;
+  // Sorted messages that are ready to be displayed in the conversation
```

Comment:
```suggestion
  /** Sorted messages that are ready to be displayed in the conversation */
```

This will allow this to be parsed as JS doc and shown in the editor ;)